HW python.py

```diff
@@ -0,0 +1,16 @@
+#my code:
+
+with open('text.txt') as file:
+    List=file.read().split()
+    wordDict={}
```

Comment:
plz follow python style guide for variable names = this one should be "word_dict", but anyway try to chose better variable names, something that describe the purpose of this var

Reply:
@AviadP   I renamed it meaningful - word_counter